pyupgrade/_main.py

```diff
@@ -591,7 +592,11 @@ def visit_Call(self, node: ast.Call) -> None:
                     if not candidate:
                         i += 1
             else:
-                self.fstrings[ast_to_offset(node)] = node
+                if (
+                    self.min_version > (3, 6) or
+                    not any(isinstance(arg, ast.Await) for arg in node.args)
```

Comment:
this is insufficient as it could be something like `1 + await f`

Reply:
gotcha - OK, will push a fix now